packages/pharos/src/components/dropdown-menu/pharos-dropdown-menu-item.scss

```diff
@@ -68,6 +68,7 @@
   margin-right: var(--pharos-spacing-one-half-x);
 }
 
+:host([disabled]),
 :host([disabled]) .dropdown-menu-item__link,
 :host([disabled]) .dropdown-menu-item__button {
```

Comment:
Are these two lines still needed?

Reply:
I think so; the other CSS properties added underneath this rule need the specificity.

Reply:
In my quick test using the Download MFE, it looks like we do need the `color` and `background-color` CSS applied at this level but the `pointer-events` for the `.dropdown-menu-item__link` and `.dropdown-menu-item__button` is no longer necessary if we have it at the `:host`. And the `color` and `background-color` have no effect at the `:host`. 

These rules may be the most accurate towards the effect they have. 
```
:host([disabled]) {
  pointer-events: none;
}

:host([disabled]) .dropdown-menu-item__link,
:host([disabled]) .dropdown-menu-item__button {
  color: var(--pharos-dropdown-menu-item-color-text-disabled);
  background-color: var(--pharos-dropdown-menu-item-color-background-disabled);
}
```

Reply:
Agreed, these should have the equivalent effect and without mixing unlike things together.

Reply:
Thanks for bringing it up @michael-iden , completely Agreed. Just committed the change.